cpp/daal/src/services/compiler/generic/env_detect_features.cpp

```diff
@@ -71,6 +71,19 @@ void run_cpuid(uint32_t eax, uint32_t ecx, uint32_t * abcd)
     #endif
 }
 
+uint32_t __daal_internal_get_max_extension_support()
+{
+    uint32_t abcd[4];
```

Comment:
clearer naming is necessary.

Reply:
The naming used is `abcd` throughout this file. This variable stores the values of the `eax, ebx, ecx, edx` registers just like the other functions. Any ideas on what to rename all of them to?